README.md

```
@@ -0,0 +1,146 @@
+f   = lambda x: np.log(np.abs(x - 0.3))
+val = vs.integrate(f, 0.0, 1.0)
+print(val)        # -1.095383...  (≈ -1.095384 within 1e-8)
+```
+
+---
+
+## 🔍 Feature Rundown
+
+| Feature                    | What it does                                        | Why you care                                  |                                            |                                 |
+| -------------------------- | --------------------------------------------------- | --------------------------------------------- | ------------------------------------------ | ------------------------------- |
+| **Auto-detect kernels**    | Spots algebraic \`                                  | r                                             | ⁻ᵝ`, logarithmic `log r\`, and mixed forms | No need to hand-label integrals |
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)